From 96703fa9b15f6d801b7d4cdb36ab0c0623c2d2b9 Mon Sep 17 00:00:00 2001
From: shangsong2 <shangsong2@lenovo.com>
Date: Mon, 5 Aug 2024 10:55:48 +0800
Subject: [PATCH] Update dsa_config_test_runner.sh
'Content-type:text/plain'

The op_config '272' test is conflict with the kernel commit 6827738dc684a merged in April 2024, it is better to remove the test.

Signed-off-by: shangsong <shangsong2@lenovo.com>
---
 test/dsa_config_test_runner.sh | 23 -----------------------
 1 file changed, 23 deletions(-)

diff --git a/test/dsa_config_test_runner.sh b/test/dsa_config_test_runner.sh
index 906856050796..59a7a290c0a2 100755
--- a/test/dsa_config_test_runner.sh
+++ b/test/dsa_config_test_runner.sh
@@ -205,29 +205,6 @@ wq_config_test()
 		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x1 -b 0x3 -c 2 "${VERBOSE}" &&
 			echo "should fail, but pass" && exit 1
 		"$ACCFG" disable-device $DSA
-
-		"$ACCFG" config-wq $DSA/$WQ0 -g 0 -m dedicated -y user -n app1 -d user -p 10 -o 272
-		"$ACCFG" config-engine $DSA/$ENG0 -g 0
-		read_ret=$(cat $IDXD_DEVICE_PATH/$DSA/$WQ0/op_config | cut -c 55-)
-		if [ "$read_ret" != "00000000,00000272" ]; then
-			echo "wq op_config 30 failed" && exit "$EXIT_FAILURE"
-		fi
-		"$ACCFG" enable-device $DSA
-		"$ACCFG" enable-wq $DSA/$WQ0
-		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x0 "${VERBOSE}" && echo "should fail, but pass" && exit 1
-		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x2 "${VERBOSE}" && echo "should fail, but pass" && exit 1
-		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x3 "${VERBOSE}" && echo "should fail, but pass" && exit 1
-		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x7 "${VERBOSE}" && echo "should fail, but pass" && exit 1
-		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x8 "${VERBOSE}" && echo "should fail, but pass" && exit 1
-		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x4 "${VERBOSE}" || echo "should pass, but fail" || exit 1
-		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x5 "${VERBOSE}" || echo "should pass, but fail" || exit 1
-		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x6 "${VERBOSE}" || echo "should pass, but fail" || exit 1
-		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x9 "${VERBOSE}" || echo "should fail, but pass" || exit 1
-		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x1 -b 0x5 -c 2 "${VERBOSE}" ||
-			echo "should pass, but fail" || exit 1
-		"$DSATEST" -w 0 -l 4096 -f 0x1 -o 0x1 -b 0x9 -c 2 "${VERBOSE}" ||
-			echo "should pass, but fail" || exit 1
-		"$ACCFG" disable-device $DSA
 	fi
 }
 
-- 
2.48.0

